### PR TITLE
Node-based Camera controller

### PR DESCRIPTION
**Pull request by stianjensen**
Check out https://github.com/stianjensen/nin-cam-controller-test for an example of how it's used

### CODE REVIEW

**Comment by stianjensen** [comment]
This should be complete and functioning now, including live reloading.

**Comment by stianjensen** [comment]
Fly-around is still missing, but I'll tackle that later.

**Comment by sigvef** [comment]
make no longer works

image

**Comment by stianjensen** [comment]
ping

File: nin/dasBoot/CameraNode.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        outputs: {
+          camera: new NIN.CameraOutput()
+        }
+      });
+
+      this.options = options;
+
+      this.cameraController = new CameraController();
+      this.outputs.camera.setValue(this.cameraController.camera);
+      Loader.loadAjax(options.path, response => this.initializeCamera(response));
```
Comment (aleksanb) [inline nin/dasBoot/CameraNode.js]: Eta reduction with `(options.path, this.initializeCamera)`?

Comment (aleksanb) [inline nin/dasBoot/CameraNode.js]: How does this loader work together with compilation?
As there won't be a file to load from then?

Comment (stianjensen) [inline nin/dasBoot/CameraNode.js]: Like any other call to loadAjax I suppose

Comment (stianjensen) [inline nin/dasBoot/CameraNode.js]: > Eta reduction with (options.path, this.initializeCamera)?

What do you mean?

Comment (aleksanb) [inline nin/dasBoot/CameraNode.js]: You can remove the lambda.

**Pull request merged**